src/i18n/en/translations.ts

```diff
@@ -34,23 +76,160 @@ export default {
+                title: "Export wallet state",
+                error_password: "Password is required",
+                error_read_file: "File read error",
+                error_no_text: "No text found in file",
+                tip: "You can export your entire Mutiny Wallet state to a file and import it into a new browser. It usually works!",
+                caveat_header: "Important caveats:",
+                caveat: "after exporting don't do any operations in the original browser. If you do, you'll need to export again. After a successful import, a best practice is to clear the state of the original browser just to make sure you don't create conflicts.",
+                save_state: "Save State As File",
+                import_state: "Import State From File",
+                confirm_replace: "Do you want to replace your state with",
+                password: "Enter your password to decrypt",
+                decrypt_wallet: "Decrypt Wallet"
+            },
+            logs: {
+                title: "Download debug logs",
+                something_screwy:
+                    "Something screwy going on? Check out the logs!",
+                download_logs: "Download Logs"
+            },
+            delete_everything: {
+                delete: "Delete Everything",
+                confirm:
+                    "This will delete your node's state. This can't be undone!",
+                deleted: "Deleted",
+                deleted_description: "Deleted all data"
+            }
+        },
+        encrypt: {},
+        lnurl_auth: {
+            title: "LNURL Auth"
+        },
+        plus: {},
+        restore: {
+            title: "Restore"
+        },
+        servers: {
+            title: "Servers",
+            caption: "Don't trust us! Use your own servers to back Mutiny."
+        }
+    },
+    swap: {
+        peer_not_found: "Peer not found",
+        channel_too_small:
+            "It's just silly to make a channel smaller than {{amount}} sats",
+        insufficient_funds: "You don't have enough funds to make this channel",
+        header: "Swap to Lightning",
+        initiated: "Swap Initiated",
+        sats_added: "sats will be added to your Lightning balance",
+        use_existing: "Use existing peer",
+        choose_peer: "Choose a peer",
+        peer_connect_label: "Connect to new peer",
+        peer_connect_placeholder: "Peer connect string",
+        connect: "Connect",
+        connecting: "Connecting...",
+        confirm_swap: "Confirm Swap"
+    },
+    error: {
+        load_time: {
+            stuck: "Stuck on this screen? Try reloading. If that doesn't work, check out the",
+            emergency_link: "emergency kit."
+        },
+        not_found: {
+            title: "Not Found",
+            wtf_paul: "This is probably Paul's fault."
+        }
+    },
     create_an_issue: "Create an issue",
-    feedback: "Bugs? Feedback?",
+    FFfeedback: "Bugs? Feedback?",
```

Comment:
typo? also where is this used?

Reply:
actually no, I added those to get rid of an error caused by the repeat `feedback` key and set myself to look for it later but never did :sweat_smile:. I don't think it is used, I think it is an artifact from before #378 and i never removed it